x-pack/solutions/observability/packages/kbn-streams-schema/src/models/ingest/base.ts

```diff
@@ -63,6 +65,7 @@ const wiredIngestSchema: z.Schema<WiredIngest> = z.intersection(
     wired: z.object({
       fields: fieldDefinitionSchema,
     }),
+    lifecycle: z.optional(ingestStreamLifecycleSchema),
```

Comment:
can we make this not optional, and use something like `lifecycle: { never: {} }`? I'm not a fan of optional properties, easy to forget, hard to type check

Reply:
Agreed, I'll update

Reply:
updated in https://github.com/elastic/kibana/pull/207083/commits/dc4714afd258690f6a4e2e55f41f35d15c922fc8